jupyter/jupyter-web-app/base/cluster-role.yaml

```diff
@@ -31,6 +31,21 @@ rules:
   - delete
   - get
   - list
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - list
```

Comment:
Isn't this just a repetition of the above?

Reply:
Sorry, didn't get it right the first time either. `jupter-web-app` should be able to "list" and `notebook-controller` should be able to "get/list/watch/delete". I mistakenly edited "jupyter/jupyter-web-app" but left "jupyter/notebook-controller". I amended both commits.

Reply:
Ok, thanks.